src/julia_to_gap.jl

```diff
@@ -175,4 +189,33 @@ function julia_to_gap(
     return record
 end
 
+## GAP objects: deal with recursion (subobjects may be Julia objects)
+function julia_to_gap(
```

Comment:
so this is for things like converting a Julia vector of GAP lists of Julia objects? Perhaps expand the comment a little bit explain that?